build.gradle

```diff
@@ -21,7 +21,7 @@ ext {
     coroutinesVersion = '1.5.2'
     androidxWorkVersion = "2.7.0"
 
-    fluxCVersion = 'trunk-69f1ea736ac0449f4724eba897980e495f0b6956'
+    fluxCVersion = '1.45.0'
```

Comment:
I just checked in my Git client, and the additional commits between commit `69f1ea7` — that WPAndroid's trunk was pointing to at the time this code freeze was done — and `1.45.0` — the commit/tag we're pointing to now — are [only FluxC commits related to changes for Woo, not WP](https://github.com/wordpress-mobile/WordPress-FluxC-Android/compare/69f1ea7...1.45.0) — namely [FluxC#2438](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2438), [FluxC#2437](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2437), [FluxC#2435](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2435) and [FluxC#2436](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2436).

So given that, we should be fine including those extra FluxC commits here in WP's code freeze. 👍